ccd-definition/AuthorisationCaseField/CareSupervision/la-barrister.json

```diff
@@ -786,7 +793,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "courtBundleTabHeading",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json